ga4gh/backends.py

```diff
@@ -280,30 +280,30 @@ def __init__(self, variantSetId, vcfPath):
         # self.tb chrom -> vcf dict
         self._tabixMap = {}
 
-        for fn in glob.glob(os.path.join(vcfPath, "*.vcf.gz")):
-            tabixfile = pysam.Tabixfile(fn)
+        for vcfFile in glob.glob(os.path.join(vcfPath, "*.vcf.gz")):
+            tabixfile = pysam.Tabixfile(vcfFile)
             for chrom in tabixfile.contigs:
                 if chrom in self._tabixMap:
                     raise Exception("cannot have overlapping VCF files.")
                 self._tabixMap[chrom] = tabixfile
 
     def convertVariant(self, record):
-        v = protocol.GAVariant()
```

Comment:
I default to saying it's ambiguous and the verbosity is limited. Particularly for people who are used to a particular shorthand that is different. For instance, above, there was a variable ```fn``` which I would assume is a reference to a function, but it's actually suppose to be filename. I don't have any associations with v, but I'm sure somebody does.

That being said, I can imagine a style proposal (that would need to be documented and voted upon, presumably) where we reserve a few one letter variables like:

variable name|type
-------------------|-----
v|GAVariant
vs|GAVariantSet

Etc, etc. This allows a controlled level of terseness and keeps everyone pretty sync'd about what names are appropriate.